code/modules/vore/eating/bellymodes_vr.dm

```diff
@@ -121,12 +121,22 @@
 			for(var/slot in slots)
 				var/obj/item/thingy = M.get_equipped_item(slot = slot)
 				if(thingy)
-					M.unEquip(thingy,force = TRUE)
-					thingy.forceMove(owner)
+					if(!M.canUnEquip(thingy))
+						continue
+					if(thingy == M.get_equipped_item(slot_w_uniform))
+						var/list/stash = list(slot_r_store,slot_l_store,slot_wear_id,slot_belt)
+						for(var/stashslot in stash)
+							var/obj/item/SL = M.get_equipped_item(stashslot)
+							if(SL)
+								SL.forceMove(src)
+								contents |= SL
```

Comment:
May call Entered twice for this item. Contents is a Byond-managed list, there's no need to ever update it manually (nor should you try)